run.py

```diff
@@ -1,7 +1,12 @@
 from tutor import app, db
+from tutor.models import model
+
+
+db.create_all()
```

Comment:
this is probably a bad Idea - what happens if you want to keep th DB between runs of the App (as you would in production... but also if you do `vagrant halt` and `vagrant up` instead of `vagrant destroy` every time).

Reply:
It keeps the DB between `vagrant halt` and `vagrant up`. `create_all()` create the not exists tables, but don't initialize it every time. It does nothing with the already exists ones.

Reply:
So what happens when you do:
1. `vagrant up`
2. `vagrant halt`
3. Change a model class
4. `vagrant up`

(That would seem to me to be a typical process for working on your models over time... A different but relats questions is wither you can write and test model code changes without shutting down and restarting the Vagrant VM....)